### PR TITLE
Chore: rename time_series2.ts->timeSeries2.ts; update references 

### PR DESCRIPTION
**Pull request by gupta437**
**What this PR does / why we need it**:
🐫 This PR refactors time_series2.ts->time_series2.ts to follow the naming guidelines for camelCasing 🐫

**Which issue(s) this PR fixes**:
[#51230](https://github.com/grafana/grafana/issues/51230)

Fixes #
Renames and updates references of time_series2 to timeSeries2.

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/features/plugins/plugin_loader.ts [hunk 1]
```diff
@@ -115,7 +115,7 @@ exposeToPlugin('app/core/utils/kbn', kbn);
 exposeToPlugin('app/core/utils/ticks', ticks);
 exposeToPlugin('app/core/config', config);
 exposeToPlugin('app/core/time_series', TimeSeries);
-exposeToPlugin('app/core/time_series2', TimeSeries);
```
Comment (ryantxu) [inline public/app/features/plugins/plugin_loader.ts:118]: 😬 this is all legacy anyway.  If we change how it is exposed, we should just remove it

Comment (torkelo) [reply public/app/features/plugins/plugin_loader.ts:118]: We should not change this part (path imported by plugins)

Comment (gupta437) [reply public/app/features/plugins/plugin_loader.ts:118]: Thankyou for the feedback! So, not change here?

**Comment by ivanahuckova** [comment]
@gupta437 In https://github.com/grafana/grafana/issues/51230 we have specified folders in which we would like to change the casing. If you plan to work on different files, please create an issue first with the list of files and get feedback from grafana team before working on them.

**Pull request closed without merging**